model_archiver/model_packaging.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Command line interface to export model files to be used for inference by MXNet Model Server
+"""
+
+import logging
+from .arg_parser import ArgParser
+from .model_packaging_utils import ModelExportUtils
+
+
+def export_model(args, manifest):
+    """
+    Internal helper for the exporting model command line interface.
+    """
+    model_path = args.model_path
+    model_name = args.model_name
+    ModelExportUtils.check_model_name_regex(model_name)
```

Comment:
It's not obvious you exit() inside the function, another reason not use sys.exit()

Reply:
Renamed the method.